app/src/main/assets/no_chart.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<title>Patient Chart</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+<style type="text/css">
+    h1 {
+        color: red;
+    }
+</style>
+<body>
+<br/><br/>
+<center>
+    <h1>Invalid profile applied</h1>
```

Comment:
Maybe add some instructions about how to apply a profile? e.g. "Log in to the server, choose 'Manage Buendia Profiles'", just so the user knows how to fix the problem?